.github/workflows/mac-build.yml

```diff
@@ -40,6 +40,8 @@ jobs:
           cp ./LICENSE build
           cp ./CHANGELOG.md build
           cd build
+          install_name_tool -id @rpath/libplay_cpp_sdk.dylib ./lib/libplay_cpp_sdk.dylib
```

Comment:
Modify the path in packing probably has a side effect: the binary was not tested in integration test.

It is better to modify the `build.rs` or setup `.cargo/config` and pass the the option to compiler: https://github.com/rust-lang/cargo/issues/5077

Reply:
ok, i'm checking now

Reply:
checked with tag, now released with @rpath like this
```
 otool -L libplay_cpp_sdk.dylib
libplay_cpp_sdk.dylib:
	@rpath/libplay_cpp_sdk.dylib (compatibility version 0.0.0, current version 0.0.0)
```